src/libsync/capabilities.cpp

```diff
@@ -190,6 +190,14 @@ PushNotificationTypes Capabilities::availablePushNotifications() const
         pushNotificationTypes.setFlag(PushNotificationType::Files);
     }
 
+    if (types.contains("activities")) {
+        pushNotificationTypes.setFlag(PushNotificationType::Activities);
+    }
+
+    if (types.contains("notifications")) {
```

Comment:
For these one you could also add tests in `test/testcapabilities.cpp`. I know these are small tests but then we get a more complete test suite :)